### DISCUSSION TITLE
README

### DISCUSSION THREAD

**Discussion by jgchoti** [HYF-Class22/agile-development-group3-hyf-redesign#40] [General]
Below is readme which I will update once we have a screenshot :) 

# HackYourFuture Belgium Redesign

> In this project, our goal is to create an improved and user-friendly home page using HTML and plain CSS. We'll focus on delivering a clean and responsive design.

## Table of contents

- [General info](#general-info)
- [Screenshots](#screenshots)
- [Technologies](#technologies)
- [Features](#features)
- [Status](#status)
- [Contact](#contact)

## General info

The purpose of this project is to redesign the HackYourFuture Belgium website. We aim to provide an appealing home page that caters to the needs of different user groups, including students, volunteers, partners and sponsors.

## Screenshots

![Example screenshot](./planning/screenshot.jpg)

## Technologies

- HTML5 for semantic structure.
- Plain CSS for styling
- GitHub for collaborating

## Features

- Responsive design
- User-specific sections for students, volunteers, partners and sponsors.

## Status

Project is: _in progress_

## Contact

By [Choti](https://github.com/jgchoti), [Haftu](https://github.com/Tesfay20), [Hitomi](https://github.com/hitomipupil),[Prince](https://github.com/rugiraprince)

**Comment by hitomipupil** [comment]
@jgchoti 
for technologies, we can add:
- GitHub for collaborating
maybe? :)

**Reply by jgchoti** [reply]
noted
